pkgs/by-name/al/alacritty-theme/package.nix

```
@@ -25,7 +25,8 @@ stdenvNoCC.mkDerivation (self: {
   sourceRoot = "${self.src.name}/themes";
   installPhase = ''
     runHook preInstall
-    install -Dt $out *.toml
+    mkdir -p $out/share/$pname
+    install -Dt $out/share/$pname *.toml
```

```suggestion
    install -Dt $out/share/alacritty-theme *.toml
```
Avoid interpolating `pname`. Changing `pname` should never alter the build output.

---

Done